src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -74,6 +74,15 @@ int32_t viewport_interaction_get_item_left(int32_t x, int32_t y, viewport_intera
     tileElement = info->tileElement;
     sprite = (rct_sprite*)tileElement;
 
+    // Allows only balloons to be popped and ducks to be quacked in title screen
+    if (gScreenFlags & SCREEN_FLAGS_TITLE_DEMO)
+    {
+        if (info->type == VIEWPORT_INTERACTION_ITEM_SPRITE && sprite->generic.sprite_identifier == SPRITE_IDENTIFIER_MISC)
+            return info->type;
+        else
+            return info->type = VIEWPORT_INTERACTION_ITEM_NONE;
```

Comment:
Not too confident about this assignment here. Should perhaps just return `VIEWPORT_INTERACTION_ITEM_NONE` instead?

Reply:
Still learning git hub here.

Ok, looking into *why* I'm making this assignment here. I'm sure I had reasoned it out. But I don't remember.

Reply:
Yup, looks like that assignment isn't necessary. Thanks. I'll fix that.

Ok, so what do I do here now? Talking about this little thread in particular. Do I click resolve/start a review? Do I wait until I've made the next commit?

Reply:
You can 'resolve' the thread/conversation after pushing a new commit, no worries.